dashboard/config/routes.rb

```diff
@@ -323,31 +326,29 @@
       get 'instructions'
     end
 
-    ## TODO: Once we move levels over to /lessons as well combine the routing rules
-    resources :lessons, only: [:show], param: 'position' do
+    # /s/xxx/lessons/yyy
+    resources :lessons, only: [:show], path: "/lessons", param: 'position', format: false do
```

Comment:
Just curious, is it necessary to specify `path` here (since it matches the resource name)?

Also, I noticed that we also have edit and update actions defined for lessons above on 297.  Is it intentional that these are separate?  (I could imagine us wanting to keep those closer to the other edit/update entries.)

Reply:
I'll try removing path since we are no longer using stage you are probably right its no longer needed. I'll also take a look at the other lesson entries to see if I can bring them closer together

Reply:
Oh I know why I left the other lessons stuff separate. Its because when you are editing or updating you are not using the nested route inside the script like we do for showing.